Dockerfile

```diff
@@ -2,11 +2,12 @@
 
 # FROM openjdk:16-alpine3.13 as spigot
 
-# RUN apk add --update --no-cache git curl
 
-# #NOTE if you have a local jar...
-# # RUN mkdir /spg
-# # COPY ./spigot-1.17.1.jar /spg/
+#Note - Curl doesnt work but WGET does??? WHY
+#RUN mkdir /tmp/sql /sql \
+#    && wget -O /tmp/sql/mysql.tar.gz \
+#    https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-8.0.26.tar.gz
```

Comment:
Move this download into setup.sh
Also download it directly into where it should go, extract it and then delete the .tar; There is no point in using /tmp
Spigot is downloaded in the same way so use that as a reference on how to write it

Reply:
Done